### ISSUE TITLE
Variables added to process.env are not removed afterwards

### ISSUE THREAD

**Issue by joebowbeer** [ashiina/lambda-local#125]
I noticed that `process.env` is not restored after a lambda execution. This complicates unit tests that invoke lambdaLocal with environment variables: variables added in one test will also be present in subsequent tests unless they are explicitly overridden in subsequent tests.

**Comment by gpotter2** [comment]
Hi !
Thanks a lot for signaling !

Could you retry with https://github.com/ashiina/lambda-local/issues/125 ?
There is now a `envdestroy` option to delete the added vars while exiting :)

**Comment by joebowbeer** [comment]
Hi. Thanks for the quick turnaround. I haven't tried it yet but it looks like it will do the trick.

I did notice by eyeballing the code that `envdestroy` doesn't undo the effect of `envfile`. I suggest you change this or clarify it in the documentation.

I don't use `envfile` in my unit tests so it doesn't matter to me.

By the way, I noticed that the lambdaLocal function documentation in the README doesn't mention the `envfile` and `envdestroy` parameters.
